commit 69e855e7b864e1bd609afd7d7dd812de60f605e5
Author: admin <admin@csa010.pne.jp>
Date:   Mon Apr 16 04:19:33 2012 +0900

    add html source routing

diff --git a/script/pms/app/AppKernel.php b/script/pms/app/AppKernel.php
index 1cdd212..2195751 100644
--- a/script/pms/app/AppKernel.php
+++ b/script/pms/app/AppKernel.php
@@ -19,6 +19,7 @@ class AppKernel extends Kernel
             new JMS\SecurityExtraBundle\JMSSecurityExtraBundle(),
             new PMS\ApiBundle\PMSApiBundle(),
             new PMS\ViewerBundle\PMSViewerBundle(),
+            new Deploy\HtmlBundle\DeployHtmlBundle(),
         );
 
         if (in_array($this->getEnvironment(), array('dev', 'test'))) {
diff --git a/script/pms/src/Deploy/HtmlBundle/Controller/DefaultController.php b/script/pms/src/Deploy/HtmlBundle/Controller/DefaultController.php
new file mode 100644
index 0000000..c954830
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/Controller/DefaultController.php
@@ -0,0 +1,15 @@
+<?php
+
+namespace Deploy\HtmlBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+
+class DefaultController extends Controller
+{
+    
+    public function indexAction()
+    {
+        return $this->redirect('/index.html');
+    }
+}
diff --git a/script/pms/src/Deploy/HtmlBundle/DependencyInjection/Configuration.php b/script/pms/src/Deploy/HtmlBundle/DependencyInjection/Configuration.php
new file mode 100644
index 0000000..a841018
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/DependencyInjection/Configuration.php
@@ -0,0 +1,29 @@
+<?php
+
+namespace Deploy\HtmlBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('deploy_html');
+
+        // Here you should define the parameters that are allowed to
+        // configure your bundle. See the documentation linked above for
+        // more information on that topic.
+
+        return $treeBuilder;
+    }
+}
diff --git a/script/pms/src/Deploy/HtmlBundle/DependencyInjection/DeployHtmlExtension.php b/script/pms/src/Deploy/HtmlBundle/DependencyInjection/DeployHtmlExtension.php
new file mode 100644
index 0000000..2d4a683
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/DependencyInjection/DeployHtmlExtension.php
@@ -0,0 +1,28 @@
+<?php
+
+namespace Deploy\HtmlBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * This is the class that loads and manages your bundle configuration
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
+ */
+class DeployHtmlExtension extends Extension
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.yml');
+    }
+}
diff --git a/script/pms/src/Deploy/HtmlBundle/DeployHtmlBundle.php b/script/pms/src/Deploy/HtmlBundle/DeployHtmlBundle.php
new file mode 100644
index 0000000..5ae69e3
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/DeployHtmlBundle.php
@@ -0,0 +1,9 @@
+<?php
+
+namespace Deploy\HtmlBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class DeployHtmlBundle extends Bundle
+{
+}
diff --git a/script/pms/src/Deploy/HtmlBundle/Resources/config/routing.yml b/script/pms/src/Deploy/HtmlBundle/Resources/config/routing.yml
new file mode 100644
index 0000000..0801779
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/Resources/config/routing.yml
@@ -0,0 +1,3 @@
+DeployHtmlBundle_homepage:
+    pattern:  /
+    defaults: { _controller: DeployHtmlBundle:Default:index }
diff --git a/script/pms/src/Deploy/HtmlBundle/Resources/config/services.yml b/script/pms/src/Deploy/HtmlBundle/Resources/config/services.yml
new file mode 100644
index 0000000..352d9db
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/Resources/config/services.yml
@@ -0,0 +1,7 @@
+parameters:
+#    deploy_html.example.class: Deploy\HtmlBundle\Example
+
+services:
+#    deploy_html.example:
+#        class: %deploy_html.example.class%
+#        arguments: [@service_id, "plain_value", %parameter%]
diff --git a/script/pms/src/Deploy/HtmlBundle/Resources/doc/index.rst b/script/pms/src/Deploy/HtmlBundle/Resources/doc/index.rst
new file mode 100644
index 0000000..e69de29
diff --git a/script/pms/src/Deploy/HtmlBundle/Resources/translations/messages.fr.xliff b/script/pms/src/Deploy/HtmlBundle/Resources/translations/messages.fr.xliff
new file mode 100644
index 0000000..fd59e6c
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/Resources/translations/messages.fr.xliff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>Symfony2 is great</source>
+                <target>J'aime Symfony2</target>
+            </trans-unit>
+        </body>
+    </file>
+</xliff>
diff --git a/script/pms/src/Deploy/HtmlBundle/Resources/views/Default/index.html.twig b/script/pms/src/Deploy/HtmlBundle/Resources/views/Default/index.html.twig
new file mode 100644
index 0000000..4ce626e
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/Resources/views/Default/index.html.twig
@@ -0,0 +1 @@
+Hello {{ name }}!
diff --git a/script/pms/src/Deploy/HtmlBundle/Tests/Controller/DefaultControllerTest.php b/script/pms/src/Deploy/HtmlBundle/Tests/Controller/DefaultControllerTest.php
new file mode 100644
index 0000000..0bb3a59
--- /dev/null
+++ b/script/pms/src/Deploy/HtmlBundle/Tests/Controller/DefaultControllerTest.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace Deploy\HtmlBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', '/hello/Fabien');
+
+        $this->assertTrue($crawler->filter('html:contains("Hello Fabien")')->count() > 0);
+    }
+}

